gps/vcs_source_test.go

```diff
@@ -16,7 +20,8 @@ func TestSlowVcs(t *testing.T) {
 	t.Run("source-gateway", testSourceGateway)
 	t.Run("bzr-repo", testBzrRepo)
 	t.Run("bzr-source", testBzrSourceInteractions)
-	t.Run("svn-repo", testSvnRepo)
+	// TODO(kris-nova) re-enable syn-repo after gps is merged into dep
```

Comment:
Why disable this one?

I'm not gonna hold up the PR for it, but noting it so we can make an issue 😄